SlipeServer.Packets/Definitions/Ped/PedSyncPacket.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using SlipeServer.Packets.Builder;
+using SlipeServer.Packets.Enums;
+using SlipeServer.Packets.Reader;
+
+namespace SlipeServer.Packets.Definitions.Ped
+{
+    public class PedSyncPacket : Packet
+    {
+        public override PacketId PacketId { get; } = PacketId.PACKET_ID_PED_SYNC;
+        public override PacketReliability Reliability { get; } = PacketReliability.ReliableSequenced;
```

Comment:
Ped sync packet should not be sent reliably, only sequenced. It is sent continously and should be able to drop packets, but they should arrive in the right order (sequenced)

```suggestion
        public override PacketReliability Reliability { get; } = PacketReliability.UnreliableSequenced;
```